scalafix-sbt/src/main/scala/scalafix/internal/sbt/JGitCompletions.scala

```diff
@@ -13,8 +13,9 @@ import java.nio.file.Path
 import scala.collection.JavaConverters._
 
 class JGitCompletion(cwd: Path) {
-  private val isGitRepository = RepositoryCache.FileKey
-    .isGitRepository(cwd.resolve(DOT_GIT).toFile, FS.DETECTED)
+  private val isGitRepository =
+    RepositoryCache.FileKey
+      .isGitRepository(cwd.resolve(DOT_GIT).toFile, FS.DETECTED)
```

Comment:
formatting

Reply:
FYI, you remove the newline after `=` and it will stay unchanged. Scalafmt sometimes respects line breaks in the original source